cinema_service/urls.py

```diff
@@ -3,6 +3,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("api/user/", include("user.urls", namespace="user")),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
Similarly, ensure that the `cinema.urls` module has an `app_name` defined, as this is required when using namespaces in Django URL configurations. Verify this in the `cinema/urls.py` file.